docs/performance/inserts/bulk.txt

```diff
@@ -93,6 +98,52 @@ Combine ``UNNEST`` with ``INSERT`` to perform bulk inserts:
     ...                       ['Arthur', 'Trillian', 'Marvin']));
     INSERT OK, 3 rows affected  (... sec)
 
+Multiple Value Expressions
+--------------------------
+
+You can insert multiple rows with multiple value expressions, like so:
+
+.. code-block:: psql
+
+    cr> INSERT INTO my_table (id, name)
+    ...      VALUES (1, 'Arthur'),
+    ...             (2, 'Trillian'),
+    ...             (2, 'Marvin');
+    INSERT OK, 3 rows affected  (... sec)
+
+This method of doing bulk inserts is just as performant as the ``UNNEST``
```

Comment:
With a large amount of values this will actually be slower because parsing the statement becomes more expensive.